client/src/components/Form/Form.scss

```diff
@@ -147,7 +147,8 @@
 .form__field-holder p.readonly,
 .form__field-holder input.readonly,
 .form__field-holder span.readonly,
-.readonly .form__field-holder > div {
+.readonly .form__field-holder > div:not(.form__field-description) {
+  pointer-events: none;
```

Comment:
```suggestion
```
I'm don't think we should be setting this to none in the context of this PR, because I really don't know what all the possible scenarios are, and I don't even know if it's desirable. I did a quick search for this and a couple of comments [here](https://ux.stackexchange.com/questions/77163/should-disabled-elements-ever-have-pointer-events) were negative about setting this to `none`

Reply:
Updated